### PR TITLE
Feature/mds 551 contact managment eor

### PR DESCRIPTION
**Pull request by nayr974**
Scope was reduced for this sprint. This is not feature complete, but pieces that are should be reviewed and accepted as part of sprint 11. Remaining work is in new Jira ticket (923).

### CODE REVIEW

File: frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  ],
+};
+
+export class ViewPermittee extends Component {
+  componentDidMount() {
+    this.props.fetchPartyRelationshipTypes();
+  }
+
+  onSubmitAddPartyRelationship = (values) => {
+    /*     this.props
```
Comment (NWCalvank) [inline frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js]: Please remove all the commented-out code in this file.

File: frontend/src/components/modalContent/AddPartyRelationshipModal.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  onSubmit: PropTypes.func.isRequired,
+  handleChange: PropTypes.func.isRequired,
+  onPartySubmit: PropTypes.func.isRequired,
+  title: PropTypes.string.isRequired,
+  partyType: PropTypes.string.isRequired,
+  parties: PropTypes.object.isRequired,
+  partyIds: PropTypes.array.isRequired,
+};
+
+const defaultProps = {
```
Comment (NWCalvank) [inline frontend/src/components/modalContent/AddPartyRelationshipModal.js]: Required props don't need defaults.

File: python-backend/app/api/parties/party/resources/manager_resource.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    parser.add_argument('mine_guid', type=str, help='Mine guid.')
+    parser.add_argument('effective_date', type=lambda x: datetime.strptime(x, '%Y-%m-%d'), help='Effective date. In the format of YYYY-MM-DD.')
+
+    @api.doc(params={'mgr_appointment_guid': 'Manager guid.'})
+    @jwt.requires_roles(["mds-mine-view"])
+    def get(self, mgr_appointment_guid):
+        manager = MgrAppointment.find_by_mgr_appointment_guid(mgr_appointment_guid)
+        if manager:
+            return manager.json()
+        return self.create_error_payload(404, 'Manager not found'), 404
```
Comment (NWCalvank) [inline python-backend/app/api/parties/party/resources/manager_resource.py]: This seems backwards. The pattern I'm used to seeing is to check for the error and fall through to the success case

File: python-backend/app/api/parties/party_appt/models/mine_party_appt_type.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+from ....utils.models_mixins import AuditMixin, Base
+
+
+class MinePartyAppointmentType(AuditMixin, Base):
+    __tablename__ = "mine_party_appt_type_code"
+    mine_party_appt_type_code = db.Column(db.String(3), primary_key=True)
+    description = db.Column(db.String(100), nullable=False)
+    display_order = db.Column(db.Integer)
+    active_ind = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
+    #relationship to all appt's with a given type?
```
Comment (NWCalvank) [inline python-backend/app/api/parties/party_appt/models/mine_party_appt_type.py]: Remove or make informative

File: frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js (hunk 1 above)
Comment (nayr974) [reply frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js]: This is WIP code that we want to carry forward.

File: python-backend/app/api/parties/party/resources/manager_resource.py (hunk 3 above)
Comment (Jsyro) [reply python-backend/app/api/parties/party/resources/manager_resource.py]: I just split the resource into their own file, but fixed

File: python-backend/app/api/parties/party_appt/models/mine_party_appt_type.py (hunk 4 above)
Comment (Jsyro) [reply python-backend/app/api/parties/party_appt/models/mine_party_appt_type.py]: WIP placeholder for me later. will be removed later.

File: frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+  closeModal: PropTypes.func.isRequired,
+  handleChange: PropTypes.func.isRequired,
+  handlePartySubmit: PropTypes.func.isRequired,
+  fetchPartyRelationshipTypes: PropTypes.func.isRequired,
+  partyRelationshipTypes: PropTypes.array.isRequired,
+};
+
+const defaultProps = {
+  expectedDocumentStatusOptions: [],
+  partyTypeOptions: [
```
[the remark this answers is not in the document]
Comment (nayr974) [reply frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js]: The value/label dictionary will be removed basically immediately and is there as a placeholder for hooking up front-end functionality while the backend is incomplete. Added proptypes.

File: frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js (hunk 1 above)
Comment (nayr974) [reply frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js]: Un-commented.

File: frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js [hunk 6]
```diff
@@ -0,0 +1,154 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import PropTypes from "prop-types";
+import { Card, Row, Col, Select } from "antd";
+import RenderSelect from "@/components/common/RenderSelect";
+
+import ConditionalButton from "@/components/common/ConditionalButton";
+import { modalConfig } from "@/components/modalContent/config";
+import * as String from "@/constants/strings";
```
[the remark this answers is not in the document]
Comment (nayr974) [reply frontend/src/components/mine/ContactInfo/ViewPartyRelationships.js]: Removed.

File: frontend/src/components/modalContent/AddPartyRelationshipModal.js (hunk 2 above)
Comment (nayr974) [reply frontend/src/components/modalContent/AddPartyRelationshipModal.js]: Removed.

**Pull request merged**